pandas/_libs/window/indexers.pyx

```diff
@@ -74,14 +76,27 @@ def calculate_variable_window_bounds(
     # right endpoint is open
     else:
         end[0] = 0
+    if center_window:
+        for j in range(0, num_values+1):
+            if (index[j] == index[0] + index_growth_sign * window_size / 2 and
```

Comment:
Can we put all this `if` statement in 1 line?

Reply:
Tried this, but seemed to be an issue with the max character count per line (pre-commit), so changed this back again now

![image](https://user-images.githubusercontent.com/52446786/106437300-6f86f980-6475-11eb-9fd0-dcfbd8152318.png)